views/update/update-trusts-tell-us-about-individual-bo.html

```diff
@@ -0,0 +1,15 @@
+{% extends "update-layout.html" %}
+
+{% set title = "Tell us about an individual (Trusts)" %}
```

Comment:
If the steer from Matthew is keep it the same as registration, we should drop "(trusts)"

Reply:
I've approved but please fix this before merging